src/components/molecules/EventModal/EventModal.tsx

```diff
@@ -39,6 +43,12 @@ const EventModal = ({
 
   const uniqueId = uuid.v4().toString();
 
+  function resetFieldTextOnEventlessDay(updatedNotes: Note[]) {
+    if (updatedNotes?.length === 0) {
```

Comment:
Suggestion: This could be:
```
if (!updatedNotes?.length) {
```

`0` is falsey and by inverting it with the `!`, it would be `true`, hence being the same as `=== 0`.